easybuild/easyblocks/n/namd.py

```diff
@@ -59,7 +59,7 @@ def configure_step(self):
 
         # complete Charm ++ and NAMD architecture string with compiler family
         comp_fam = self.toolchain.comp_family()
-        if self.toolchain.options['usempi']:
+        if 'usempi' in self.toolchain.options and self.toolchain.options['usempi']:
```

Comment:
I'd rather go with something like:

```python
if self.toolchain.options.get('usempi', False):
```

which is already being used in other easyblocks